### ISSUE TITLE
Freetype Host doesn't seem to have the freetype repo mirrored anymore.

### ISSUE THREAD

**Issue by AElimrani** [rougier/freetype-py#200]
The default FREETYPE_HOST in setup-build-freetype.py seems to not host the file anymore. https://mirrors.sarata.com/non-gnu/freetype/

**Comment by AElimrani** [comment]
probably temporary though it seems to have been down everytime I have tried accessing it in recent days

**Comment by HinTak** [comment]
This is a bit wrong - it should use one of the official places, which is savannah, gitlab.freedesktop, or github. Cc @madig why did you use the odd sarata mirror?

**Comment by HinTak** [comment]
There is an official mirror here, https://github.com/freetype/freetype, which links to new gitlab freedesktop place.

**Comment by AElimrani** [comment]
Yes this is the reason for originally opening the issue; the mirror was not familiar and it was not working but then I figured that downtime isn't worthy of an issue and the uncommon mirror might have been chosen for a reason. Since the FREETYPE_HOST seems to be used to get binaries, I just changed it locally to this: https://download.savannah.gnu.org/releases/freetype/ . But looking at other mirrors it does seem like freetype was not available everywhere (the GNU mirrors at berkeley, uwaterloo and others don't seem to host freetype releases or code, so maybe that's why the current mirror was chosen? )

**Comment by HinTak** [comment]
Freedesktop is the current official upstream, and github is also an official mirror monitored/maintained by upstream people. Hence I am asking @madig why he chose that particular one (and also not using one of the others, even if savannah may be unreliable).

**Comment by AElimrani** [comment]
Ah you are right, I thought it was also used to get compiled binary packages (not available on gitlab), but it isn't.

**Comment by madig** [comment]
I don't remember and it's possible I just grabbed the first host I saw. Change as needed.
